src/components/AddNote.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import { Icon } from 'semantic-ui-react';
+import { mainStateDispatchContext } from './Main';
+
+const AddNoteWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: black;
```

Comment:
use #333. It is much more readable that this "black"

Reply:
Done